qemu/tests/ksm_overcommit.py

```diff
@@ -642,8 +642,8 @@ def split_parallel():
     logging.debug(utils_test.get_memory_info(lvms))
 
     # Copy ksm_overcommit_guest.py into guests
-    shared_dir = os.path.dirname(data_dir.get_data_dir())
-    vksmd_src = os.path.join(shared_dir, "scripts", "ksm_overcommit_guest.py")
+    vksmd_src = os.path.join(data_dir.get_shared_dir(),
+                             "scripts/ksm_overcommit_guest.py")
```

Comment:
Yes, agree with @XueqiangWei .
And confirm with @yanan-fu , she agrees too.
The first one is better. :)